Packs/DevSecOps/Integrations/GitLab/GitLab.py

```diff
@@ -241,6 +274,118 @@ def get_version_command(client, args):
     return command_results
 
 
+def gitlab_pipelines_schedules_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Returns pipeline schedules corresponding to given arguments.
+    Args:
+        client (Client): Client to perform calls to GitLab services.
+        args (Dict[str, Any]): XSOAR arguments:
+            - 'project_id' (Required): Project ID to retrieve pipeline schedules from.
+            - 'pipeline_schedule_id': ID of specific pipeline schedule to retrieve its details.
+
+    Returns:
+        (CommandResults).
+    """
+    project_id = args.get('project_id', '')
+    pipeline_schedule_id = args.get('pipeline_schedule_id')
+    response = client.get_pipeline_schedules_request(project_id, pipeline_schedule_id)
+    response = response if isinstance(response, list) else [response]
+    outputs = [{k: v for k, v in output.items() if k in PIPELINE_SCHEDULE_FIELDS_TO_EXTRACT} for output in response]
+
+    return CommandResults(
+        outputs_prefix='GitLab.PipelineSchedule',
+        outputs_key_field='id',
+        outputs=outputs,
+        raw_response=response,
+        readable_output=tableToMarkdown('GitLab Pipeline Schedules', outputs)
```

Comment:
please use removeNull, headers and headerTransform args in all tableToMarkdown calls

Reply:
@dantavori How exactly do you want me to use headerTransform in the table to mark down?